pilot/prompts/dev_ops/ran_command.prompt

```diff
@@ -1,7 +1,8 @@
 {{ additional_info }}I ran the command `{{ command }}`
-{%- if exit_code is number %}, the exit code was {{ exit_code }}{% endif %}
 {%- if error_response %}, it {{ error_response }}{% endif %}. The output was:
 
 {{ cli_response }}
 
 If the command was successfully executed, respond with `DONE`. If it wasn't, respond with `NEEDS_DEBUGGING`.
+
+Do not respond with anything other than these two keywords.
```

Comment:
This will have to be fixed properly https://github.com/Pythagora-io/gpt-pilot/issues/188